src/PrestaShopBundle/Resources/views/Admin/Sell/Catalog/Product/Blocks/footer.html.twig

```diff
@@ -0,0 +1,46 @@
+{#**
+ * Copyright since 2007 PrestaShop SA and Contributors
+ * PrestaShop is an International Registered Trademark & Property of PrestaShop SA
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.md.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/OSL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to https://devdocs.prestashop.com/ for more information.
+ *
+ * @author    PrestaShop SA and Contributors <contact@prestashop.com>
+ * @copyright Since 2007 PrestaShop SA and Contributors
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ *#}
+{% block product_footer %}
+  <div class="product-footer justify-content-md-center">
+    <div class="col-lg-4">
+      {% if null != productId %}
+      <a
+        href="{{ path('admin_product_unit_action', {action: 'delete', id: productId}) }}"
+        class="tooltip-link btn btn-lg delete"
+        data-toggle="pstooltip"
+        id="product_form_delete_btn"
+        title="{{ 'Permanently delete this product.'|trans({}, 'Admin.Catalog.Help') }}"
+        data-placement="left"
+        data-original-title="Delete"
+      >
+        <i class="material-icons">delete</i>
+      </a>
```

Comment:
I think we should make this reusable somewhere, It looks like you repeat the whole code on every pages, while it could be a simple include with props

Reply:
oh yeah definitely, it's an ugly copy/paste from this file https://github.com/PrestaShop/PrestaShop/blob/develop/src/PrestaShopBundle/Resources/views/Admin/Product/ProductPage/Blocks/footer.html.twig

I cleaned it a little to keep the minimum because this PR is only here to have a basic integration, but we will have to improve the whole integration and layout The purpose here is just to have the CQRS and partial update handling So I'm not gonna improve this one right now